docs/topics/whatsnew/whatsnew-eap.md

```diff
@@ -108,8 +251,203 @@ kotlin {
+
+The new feature is designed to replace the `-Xallocator=std` compiler option that enabled the system memory allocator instead of the default one. Now you can disable buffering (paging of allocations) without switching memory allocators.
+
+The feature is currently [Experimental](components-stability.md#stability-levels-explained).
+To enable it, set the following option in your `gradle.properties` file:
+
+```none
+kotlin.native.binary.pagedAllocator=false
+```
+
+Please report any problems to our issue tracker [YouTrack](https://kotl.in/issue).
+
+### LLVM update from 16 to 19
+
+In Kotlin %kotlinEapVersion%, we updated LLVM from version 16 to 19.
+The new version includes performance improvements, bug fixes, and security updates.
+
+This update shouldn't affect your code, but if you encounter any issues, please report them to our [issue tracker](http://kotl.in/issue).
+
+## Kotlin/Wasm: wasmJs target separated from js target
+
+Before, the `wasmJs` target shared the same infrastructure as the `js` target, meaning that both were hosted in the same
+directory (`build/js`) and used the same NPM tasks and configurations.
+
+Now, the `wasmJs` target has its own infrastructure separate from the `js` target. This allows the
+Wasm tasks and types to be distinct from the JavaScript ones, enabling independent configuration.
+
+Additionally, the Wasm-related project files and NPM dependencies are now in a separate directory: `build/wasm`.
+
+New NPM-related tasks have been introduced for Wasm, while existing JavaScript tasks are now dedicated only to JavaScript:
+
+| **Wasm tasks**         | **JavaScript tasks** |
+|------------------------|----------------------|
+| `kotlinWasmNpmInstall` | `kotlinNpmInstall`   |
+| `wasmRootPackageJson`  | `rootPackageJson`    |
+
+Similarly, new Wasm-specific declarations are introduced:
+
+| **Wasm declarations**     | **JavaScript declarations** |
+|---------------------------|-----------------------------|
+| `WasmNodeJsRootPlugin`    | `NodeJsRootPlugin`          |
+| `WasmNodeJsPlugin`        | `NodeJsPlugin`              |
+| `WasmYarnPlugin`          | `YarnPlugin`                |
+| `WasmNodeJsRootExtension` | `NodeJsRootExtension`       |
+| `WasmNodeJsEnvSpec`       | `NodeJsEnvSpec`             |
+| `WasmYarnRootEnvSpec`     | `YarnRootEnvSpec`           |
+
+This change allows you to work with the Wasm target independently of the JavaScript target, simplifying the configuration.
+
+You don't require additional setup; this change is enabled by default.
+
+## Kotlin/JS
+
+### Fix for copy() in @JsPlainObject interfaces
+
+Kotlin/JS has an experimental plugin called `js-plain-objects`, which introduced a `copy()` function for interfaces annotated with `@JsPlainObject`. The `copy()`  function is useful for manipulating objects.
```

Comment:
```suggestion
Kotlin/JS has an experimental plugin called `js-plain-objects`, which introduced a `copy()` function for interfaces annotated with `@JsPlainObject`. The `copy()` function is useful for manipulating objects.
```

Reply:
Why is it useful? Can we say:

You can use the copy() function to manipulate objects.